src/pspm_extract_segments.m

```diff
@@ -255,14 +255,14 @@
   marker_data = {};
   if manual_chosen == 1
     for i=1:numel(data_fn)
-      [sts, ~, data] = pspm_load_data(data_fn{i}, channel{i});
+      [sts, data] = pspm_load_channel(data_fn{i}, channel{i});
```

Comment:
If my understanding is correct, the last argument of `pspm_load_channel` is optional? If so, maybe add some description to the help text of `pspm_load_channel`? If not, maybe this sentence needs to be completed?